config/pin_base.mpb

```diff
@@ -34,7 +36,7 @@ project {
   }
 
   specific (gnuace, make) {
-    compile_flags += -O3 -fomit-frame-pointer -fno-strict-aliasing -Wno-unknown-pragmas -fno-stack-protector -fPIC
+    compile_flags += -O3 -funwind-tables -fno-stack-protector -fasynchronous-unwind-tables -fomit-frame-pointer -fno-strict-aliasing -fno-rtti -faligned-new -fPIC -isystem $(PIN_ROOT)/extras/stlport/include -isystem $(PIN_ROOT)/extras/libstdc++/include -isystem $(PIN_ROOT)/extras/crt/include -isystem $(PIN_ROOT)/extras/crt/include/arch-x86_64 -isystem $(PIN_ROOT)/extras/crt/include/kernel/uapi -isystem $(PIN_ROOT)/extras/crt/include/kernel/asm-x86_64 -I$(PIN_ROOT)/source/include/pin -I$(PIN_ROOT)/source/include/pin/gen -I$(PIN_ROOT)/extras/components/include -I$(PIN_ROOT)/extras/xed-intel64/include/xed -nostdlib -fpermissive -Wno-error=all -fno-exceptions
```

Comment:
You add directories which look to be already in the intel64 feature, see https://github.com/DOCGroup/MPC/blob/b3f6d22dcb39d125475651b580e9a5805767b6a1/config/pin_base.mpb#L91. The generic part of pin_base shouldn't have architecture specifc includes

Reply:
moved into feature and took out duplicates.